README.md

```diff
@@ -1,4 +1,30 @@
+
 # Databox Container Manager
 
 Databox container manager and dashboard user interface part of the databox platform 
-see [the main repository](https://github.com/me-box/databox) for more information.
+see [the main repository](https://github.com/me-box/databox) for more information. 
+
+For developing Databox core components- Databox exposes following functions:
+1. `setHttpsHelper(helper)'
+2. `install(sla)`
+3. `uninstall(service)`
+4. `restart(container)`
+5. `connect()`
+5. `listContainers()`
+6. `generateArbiterToken(name)` this function generates token to be passed to arbitor for the service.
+7. `updateArbiter(data)` this function updates arbitor endpoint:/cm/upsert-container-info using post 'data'
```

Comment:
/cm/upsert-container-info is part of the arbiter API and only the container manager has access to it.